src/world/GameWorld.cpp

```diff
@@ -85,9 +84,6 @@ bool GameWorld::LoadMap(const std::string& mapFilePath, const std::string& luaFi
     }
 
     CreateTradeGraphs();
-
-    GetTerrainRenderer().GenerateOpenGL(*this);
-
```

Comment:
During creation of `GameWorldViewer`: https://github.com/Return-To-The-Roots/s25client/pull/471/files#diff-e660a654c4f718d425de16c0f5d5ececR34
And that one is constructed  in `dskGameInterface`.

Checking error codes showed an GL_OUT_OF_MEMORY in the first call of `glDrawArrays` when drawing the textures (only when using VBO)